src/pages/ContractDetails.vue

```diff
@@ -92,7 +92,7 @@
               </template>
             </Property>
             <Property id="code">
-              <template v-slot:name>Code</template>
+              <template v-slot:name>Contract Creation Code</template>
```

Comment:
Would prefer it called `Initcode` to align with both [HAPI](https://github.com/hashgraph/hedera-protobufs/search?q=initcode) and Ethereum standards as called out by Danno in a [HIP discussion](https://github.com/hashgraph/hedera-improvement-proposal/pull/513#discussion_r899646293). Hedera should have consistent terminology across its applications. We would rename the REST API field if we could.

Reply:
I agree, just maybe `Initcode` is a bit too much _developer-oriented_ for an explorer web interface. What do you think about `Initialization Code`?

Reply:
I think any one looking at hex data on a UI is probably a developer ;). Still prefer `Initcode` so everyone is in alignment. Perhaps a tooltip can clarify further.